roofit/roofitcore/test/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@
 # @author Patrick Bos, NL eScience Center, 2018
 
 ROOT_ADD_GTEST(simple simple.cxx LIBRARIES RooFitCore)
-ROOT_ADD_GTEST(testRooCacheManager testRooCacheManager.cxx LIBRARIES RooFitCore)
+ROOT_ADD_GTEST(testRooCacheManager testRooCacheManager.cxx LIBRARIES RooFitCore RooFit)
```

Comment:
CMake should handle everything transitively (the core dependency in this case). In the future, use
`ROOT_ADD_GTEST(testRooCacheManager testRooCacheManager.cxx LIBRARIES RooFit)`
and report if you see problems.

Reply:
Okay, I will update it in this PR.